### PR TITLE
Several minor updates

### PR DESCRIPTION
**Pull request by drobnikj**

### CODE REVIEW

File: src/consts.js [hunk 1]
```diff
@@ -108,6 +108,18 @@ const DATASET_OUTPUT_FIELDS = [
     { key: 'itemsFileUrls', label: 'Items file URLs', type: 'string' },
 ];
 
+const KEY_VALUE_STORE_SAMPLE = {
+    keyValueStore: {
+        id: '98e7lEimuGBDvFfcM',
+        name: 'zapier-test',
```
Comment (mtrunkat) [inline src/consts.js]: If this is shown in the UI then I would use name `my-store` or `my-key-value-store` rather ten `xxx-test`

File: src/consts.js [hunk 2]
```diff
@@ -108,6 +108,18 @@ const DATASET_OUTPUT_FIELDS = [
...
+        userId: 'GWnltczwPrB3uTMal',
+        createdAt: '2019-04-24T09:50:49.930Z',
+        modifiedAt: '2019-05-31T11:56:34.472Z',
+        accessedAt: '2019-05-31T11:56:33.300Z',
+    },
+    keyValueStoreRecordUrl: 'https://api.apify.com/v2/key-value-stores/56e7hEimwGKOvFfcM/records/dcfohu0g4a',
```
Comment (mtrunkat) [inline src/consts.js]: And some better key like `my-record`

File: src/consts.js [hunk 3]
```diff
@@ -108,6 +108,18 @@ const DATASET_OUTPUT_FIELDS = [
     { key: 'itemsFileUrls', label: 'Items file URLs', type: 'string' },
 ];
 
+const KEY_VALUE_STORE_SAMPLE = {
```
Comment (jancurn) [inline src/consts.js]: This would deserve a comment, what is it, which account it that etc.

File: src/creates/set_value.js [hunk 4]
```diff
@@ -31,7 +31,7 @@ module.exports = {
     noun: 'Key-Value Store Value',
     display: {
         label: 'Set Key-Value Store Record',
-        description: 'Saves a record to a key-value store.',
+        description: 'Sets a new or updates an existing record to a key-value store.',
```
Comment (jancurn) [inline src/creates/set_value.js]: This text doesn't sound right, how about:

```Creates a new or updates an existing record in a key-value store.```

**Pull request merged**